_notes/why_must_i_learn_this.md

```diff
@@ -53,6 +53,10 @@ That’s why I believe that the future of education needs to better address the
 
 # Two parters
 
+{% margin %}
+"Context is worth 80 IQ points" ~ Alan Kay
```

Comment:
nothing to add here very based

![image](https://media1.tenor.com/m/FMacCdHxgMgAAAAC/call-from-based-department.gif)